### PR TITLE
Update razor-pages-start.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fix copy/paste error

### CODE REVIEW

File: aspnetcore/tutorials/razor-pages-mac/razor-pages-start.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ From Visual Studio, select **File > Open**, and then select the *RazorPagesMovie
 
 ### Launch the app
 
-In Visual Studio, select **Run > Start Without Debugging** to launch the app. Visual Studio starts [IIS Express](https://docs.microsoft.com/iis/extensions/introduction-to-iis-express/iis-express-overview), launches a browser, and navigates to `http://localhost:5000`.
+In Visual Studio, select **Run > Start Without Debugging** to launch the app. Visual Studio starts [Kestrel](xref:fundamentals/servers/kestrel), launches a browser, and navigates to `http://localhost:5000`.
```
Comment (guardrex) [inline aspnetcore/tutorials/razor-pages-mac/razor-pages-start.md]: They call for restricting the bold text to only the UI elements. Also, that menu is actually "Debug," so use ...

> \*\*Debug\*\* > \*\*Start Without Debugging\*\*

Just as a side note: Both of those statements are true. IIS Express *is* fired up (with or without debugging). Of course, Kestrel is as well. It's fine to say that Kestrel is started by VS (indirectly).

Comment (Rick-Anderson) [reply aspnetcore/tutorials/razor-pages-mac/razor-pages-start.md]: IIS Express is fired up on a mac?

Comment (Rick-Anderson) [reply aspnetcore/tutorials/razor-pages-mac/razor-pages-start.md]: Did you check the UI on the mac?

Comment (guardrex) [reply aspnetcore/tutorials/razor-pages-mac/razor-pages-start.md]: Oh! `razor-pages-mac` ... now I see why you said "Fix copy/paste error."

My circa-2009 Mac Mini is pretty much dead. :cry: Did they name that menu "Run" on VS for Mac?

Comment (guardrex) [reply aspnetcore/tutorials/razor-pages-mac/razor-pages-start.md]: Guess so ...

capture

**Pull request merged**